src/main/java/org/killbill/billing/plugin/dwolla/core/DwollaServlet.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright 2014 Groupon, Inc
+ *
+ * Groupon licenses this file to you under the Apache License, version 2.0
+ * (the "License"); you may not use this file except in compliance with the
+ * License.  You may obtain a copy of the License at:
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.killbill.billing.plugin.dwolla.core;
+
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import org.killbill.billing.plugin.core.PluginServlet;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class DwollaServlet extends PluginServlet {
+
+    private static final Logger logger = LoggerFactory.getLogger(DwollaServlet.class);
+
+    @Override
+    protected void doGet(final HttpServletRequest req, final HttpServletResponse resp) throws ServletException, IOException {
+    }
+
+    @Override
+    protected void doPost(final HttpServletRequest req, final HttpServletResponse resp) throws ServletException, IOException {
+        final String formUrlEncoded = getRequestData(req);
```

Comment:
You probably don't need this? It looks like the Adyen test handler for HPPs query parameters encoding.